### ISSUE TITLE
logspace?

### ISSUE THREAD

**Issue by jonathanstrong** [bluss/itertools-num#4]
You don't happen to have a logspace implementation lying around anywhere, do you?

**Comment by bluss** [comment]
Nope, but it should be easy to write. This crate is looking for adoption :)
